.github/workflows/publish_to_pypi.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       uses: pypa/gh-action-pypi-publish@master
       with:
         user: __token__
-        password: ${{ secrets.PYPI_TOKEN }}
+        password: ${{ secrets.PYPI_TOKEN }}
```

Comment:
Could you add an extra line here please?

Reply:
Done